packages/@o3r/dev-tools/src/cli/set-version.ts

```diff
@@ -26,6 +35,8 @@ program
   .option('-v, --verbose', 'Display debug logs')
   .action((version: string) => {
     const cleanVersion = clean(version);
+    logger.warn(`This CLI is deprecated, please use ${bold('set-version')} from the package ${bold('@o3r/workspace')}`);
```

Comment:
@matthieu-crouzet found that there is no guarantee which bin will be used by the package manager - maybe you should discuss?

Reply:
Good time to prefix the with `o3r-` indeed